File: ce/snippets/csharp/CRMV8/auditing/cs/auditing5.cs
// <snippetauditing5>


 // Retrieve the audit history for the account and display it.
 RetrieveRecordChangeHistoryRequest changeRequest = new RetrieveRecordChangeHistoryRequest();
 changeRequest.Target = new EntityReference(Account.EntityLogicalName, _newAccountId);

 RetrieveRecordChangeHistoryResponse changeResponse =
     (RetrieveRecordChangeHistoryResponse)_service.Execute(changeRequest);

 AuditDetailCollection details = changeResponse.AuditDetailCollection;

// </snippetauditing5>

File: ce/customer-engagement/on-premises/snippets/visualbasic/CRMV8/auditingvb/vb/auditing5.vb
' <snippetauditing5>



 ' Retrieve the audit history for the account and display it.
 Dim changeRequest As New RetrieveRecordChangeHistoryRequest()
 changeRequest.Target = New EntityReference(Account.EntityLogicalName, _newAccountId)

 Dim changeResponse As RetrieveRecordChangeHistoryResponse = CType(_service.Execute(changeRequest), RetrieveRecordChangeHistoryResponse)

 Dim details As AuditDetailCollection = changeResponse.AuditDetailCollection

' </snippetauditing5>